test/e2e/e2epipeline/pipeline_e2e_test.go

```diff
@@ -12,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package e2e
+package e2epipeline
```

Comment:
Why not naming the package `pipeline` (under `e2e`) ?

Reply:
Sure, my original thought was to avoid any confusion with other pipeline package, but I agree and changed it.